### PR TITLE
Allow release package to determine what needs built

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: release_package.yml [hunk 1]
```diff
@@ -10,8 +10,30 @@
...
+        copr_package_info: "{{ copr_package_info.stdout | from_json }}"
+
+    - set_fact:
+        copr_package_version: "{{ copr_package_info['latest_build']['pkg_version'] }}"
+
+    - debug:
+        msg: "Git version:  {{ git_package_version }} \nCopr version: {{ copr_package_version }}"
+
+    - block:
+        - include: tasks/setup_sources.yml
```
Comment (ekohl) [inline release_package.yml]: Isn't plain ```include``` now deprecated?

**Comment by ehelms** [comment]
Updated the deprecated includes. And yes, we should catch release/version updates. I will add that to the job

File: release_package.yml [hunk 2]
```diff
@@ -10,8 +10,30 @@
...
+        git_package_version: "{{ git_package_info.stdout }}"
+
+    - set_fact:
+        copr_package_info: "{{ copr_package_info.stdout | from_json }}"
+
+    - set_fact:
+        copr_package_version: "{{ copr_package_info['latest_build']['pkg_version'] }}"
+
+    - debug:
+        msg: "Git version:  {{ git_package_version }} \nCopr version: {{ copr_package_version }}"
```
Comment (evgeni) [inline release_package.yml]: there is a double space before `{{ git_package_version }}`

File: release_package.yml (hunk 1 above)
Comment (evgeni) [reply release_package.yml]: It is, but the plabooks are based on other playbooks that predate the deprecation. ;)

File: release_package.yml [hunk 3]
```diff
@@ -10,8 +10,30 @@
   any_errors_fatal: false # don't bomb out the entire playbook if one host (i.e. package) fails
   gather_facts: no
   tasks:
-    - include: tasks/setup_sources.yml
+    - name: 'Get latest package info from copr'
+      command: "copr-cli get-package @theforeman/tfm-ror51 --name {{ inventory_hostname }} --with-latest-build"
```
Comment (evgeni) [inline release_package.yml]: What if there is no such puckage in the copr repo yet?

File: release_package.yml (hunk 2 above)
Comment (ehelms) [reply release_package.yml]: That is actually intentional, mostly just to align for better visual inspection.

File: release_package.yml (hunk 3 above)
Comment (ehelms) [reply release_package.yml]: Touche sir.

**Comment by ehelms** [comment]
This now ensures things not existing in copr already still get built.

**Pull request merged**